### ISSUE TITLE
What is wrong with my code?

### ISSUE THREAD

**Issue by olafghanizadeh** [ThemeFuse/Unyson#2331]
Hello,

I'm trying to do some small modifications to a Unyson built theme that I bought. I'm trying to add a list of icons to a content block depending on the Hotel Room features. I looked at the code in the other content blocks of the theme and modified it accordingly. It seems to work fine on backend and I can add icons/text, but I can not get it to print on front end. My code doesn't produce any errors now, it just doesn't show up. When I'm testing with creating a simpler field without "addable popup"-array it works and I'm able to get the value on front end.

I copied the custom content block from the parent theme and put it in the child theme. 

Here is my options.php:
http://pastebin.com/bBs8VRPj

And here is my view.php
http://pastebin.com/JMvjKTbt

Would love to get some explanation on what I'm doing wrong. Thanks!

**Comment by thednp** [comment]
This is a public development portal, for customer support please refer at developer via his/their support chanels.

**Comment by mowshon** [comment]
http://pastebin.com/JMvjKTbt
on line: 4

You can use debugging function `fw_print( $atts );` to see the array structure and what output you have. In `foreach` body you can use this too, `fw_print( $rooms )`. Maybe your PHP configuration ignore the notice error and some key from array is wrong.

**Comment by olafghanizadeh** [comment]
Hello!

I did found out the error in my code.

On line 34 I was trying to call the data for the icon set from the first loop like so:

` <?php foreach ($atts['regal_room'] as $rooms):?>
`

When it should have been like this:

`                             <?php foreach ($rooms['regal_room_facilities'] as $room_icons):?>
`

I changed the rest of the code accordingly and now it works.

However. Playing with this theme and unyson I must say that I feel like the framework seems kind of slow? Has anyone else noticed this as well?
